pkg/logger/logger.go

```diff
@@ -54,3 +54,20 @@ func (logger *BaseLogger) CopyTo(dst Logger) error {
 	_ = dst
 	return ErrCopyToNotImplemented
 }
+
+type EmptyLogger struct{}
+
+func NewEmpty() EmptyLogger {
+	return EmptyLogger{}
+}
+
+func (*EmptyLogger) AsWriter() io.Writer {
+	return nil
+}
+
+func (*EmptyLogger) LogEvent(eventName EventType, event interface{}) error { //nolint:revive // causes gopy to fail
+	return nil
+}
+func (*EmptyLogger) CopyTo(dst Logger) error { //nolint:revive // causes gopy to fail
```

Comment:
Nit: inconsistent formatting
```suggestion

func (*EmptyLogger) CopyTo(dst Logger) error { //nolint:revive // causes gopy to fail
```